nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/c2/C2NifiClientService.java

```diff
@@ -155,7 +136,7 @@ private C2ClientConfig generateClientConfig(NiFiProperties properties) {
                 C2NiFiProperties.C2_DEFAULT_CALL_TIMEOUT), TimeUnit.MILLISECONDS))
             .c2Url(properties.getProperty(C2NiFiProperties.C2_REST_URL_KEY, ""))
             .c2RequestCompression(properties.getProperty(C2NiFiProperties.C2_REQUEST_COMPRESSION_KEY, C2NiFiProperties.C2_REQUEST_COMPRESSION))
-                .confDirectory(properties.getProperty(C2NiFiProperties.C2_CONFIG_DIRECTORY_KEY, DEFAULT_CONF_DIR))
+            .confDirectory(properties.getProperty(C2NiFiProperties.C2_CONFIG_DIRECTORY_KEY, DEFAULT_CONF_DIR))
```

Comment:
The asset directory shouldn't be added to the C2ClientConfig? I mean if we treat it as a c2 parameter I think it would make sense. What do you think?

Reply:
Good point, fixed it, it also became cleaner